build/Version.props

```diff
@@ -0,0 +1,124 @@
+  </PropertyGroup>
+  
+  <PropertyGroup>  
+	<!--NuGet Properties-->
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
+    <IncludeBuildOutput>False</IncludeBuildOutput>
+	
+	<PackageName>$(MSBuildProjectName)</PackageName>
+	<Title>$(MSBuildProjectName)</Title>
+	<Version>$(PackageVersion)</Version>
+    <Description>This package contains MSBuild targets and tasks for building and publishing ASP.NET Core projects. This package is required for building and publishing ASP.NET Core projects.</Description>
+	
+    <Authors>Microsoft</Authors>
+    <ProjectUrl>https://github.com/aspnet/websdk</ProjectUrl>
+    <LicenseUrl>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</LicenseUrl>
+    <ImageUrl>http://go.microsoft.com/fwlink/?LinkID=288859</ImageUrl>
+    <Tags>WebSdk</Tags>
+    <Company>Microsoft</Company>
+    <Copyright>Copyright © Microsoft Corporation</Copyright>	
+    <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
+	
+    <GenerateAssemblyInfo Condition="'$(BuildingInsideVisualStudio)' != ''">False</GenerateAssemblyInfo>
+    <GenerateAssemblyFileVersionAttribute>False</GenerateAssemblyFileVersionAttribute>
+    <GenerateAssemblyVersionAttribute>False</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyInformationalVersionAttribute>False</GenerateAssemblyInformationalVersionAttribute>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>$(WebSdkRoot)\tools\Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ArtifactsDir>$(WebSdkRoot)\artifacts\</ArtifactsDir>
+    <GitInfoFile>$(ArtifactsDir)GitInfo.cs</GitInfoFile>
+    <GitInfoProps>$(ArtifactsDir)GitInfo.props</GitInfoProps>
+  </PropertyGroup>
+  
+  <Target Name="CollectGitInfo">
+    <PropertyGroup>
+      <GitInfoCommitCount Condition="'$(COMMIT_HASH)' != ''">$(COMMIT_COUNT)</GitInfoCommitCount>
+      <GitInfoCommitHash Condition="'$(COMMIT_HASH)' != ''">$(COMMIT_HASH)</GitInfoCommitHash>
+      <GitExitCode>0</GitExitCode>
+    </PropertyGroup>
+
+    <Exec Command="git rev-list --count HEAD" ConsoleToMSBuild="true" Condition="'$(COMMIT_HASH)' == ''" IgnoreExitCode="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitCount" />
+      <Output TaskParameter="ExitCode" PropertyName="GitExitCode" />
+    </Exec>
+
+    <Exec Command="git rev-parse HEAD" ConsoleToMSBuild="true" Condition="'$(COMMIT_HASH)' == ''" IgnoreExitCode="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitHash" />
+      <Output TaskParameter="ExitCode" PropertyName="GitExitCode" />
+    </Exec>
+
+    <PropertyGroup Condition="'$(GitExitCode)' != '0'">
+      <GitInfoCommitCount>0</GitInfoCommitCount>
```

Comment:
Is this actually useful to have? AspNetCore doesn't add this, commit hash is usually sufficient

Reply:
Templating adds it and I think cli also adds it.